packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -387,7 +387,7 @@ export default class QuanticGeneratedAnswer extends LightningElement {
   }
 
   get generatedAnswerFooterCssClass() {
-    return `slds-grid slds-grid_align-spread generated-answer__footer--${
+    return `slds-grid slds-grid_align-spread slds-wrap generated-answer__footer--${
```

Comment:
This unfortunately would break how we kinda want to display the citations. we want the citations to be wrapped in multiple lines when necessary, however if we give 'slds-wrap' to container holding the citations and the rephrase buttons it means that the citations will not  really need to be wrapped since the rephrase buttons will be them wrapped and moved to the next line.

My solution for this is just to use the public property 'multilineFoolter'.
If a user wants to use the generated answer component in a mobile interface, he just needs to set `multilineFooter` to `true` that's the purpose of this property, to solve this overflow issues.

Reply:
ok I will scratch that fix, but I tried and setting the multilineFooter property to true creates overflow on the rephrase buttons for some of the mobile resolutions (ie : iphone SE and other smaller mobile screen sizes)
See:
<img width="352" alt="Screenshot 2023-12-04 at 4 30 13 PM" src="https://github.com/coveo/ui-kit/assets/73316533/7dc0d588-9f77-4f9c-b772-4516499526f9">

To fix this, I suggest adding `slds-wrap` class to the rephrase buttons container itself.

See:
<img width="314" alt="Screenshot 2023-12-04 at 4 31 47 PM" src="https://github.com/coveo/ui-kit/assets/73316533/ba7d3184-155b-4878-b7fb-66b24694fc67">

Reply:
Unfortunately this also  has an impact that we do not really want. For example:

In the following case we do not really want to display the rephrase buttons in multiple lines, we always want to display them in the same line: 

https://github.com/coveo/ui-kit/assets/86681870/ad7b09b6-0db7-4f2b-89d1-23e09f4248ac



While very small phones are not really our main concern the ideal solution to fix that case I think is to make the rephrase buttons display only the icons and not the labels, this would liberate more space and would allow to display the rephrase buttons in one line.

Reply:
will leave it as is then since the multilineFooter option is good enough for now